site/_includes/partials/showcase-card.njk

```diff
@@ -32,6 +32,9 @@
         {{ 'i18n.common.launch_app' | i18n(locale) }}
```

Comment:
I wonder if the CTA should also change to "Download app" in this case...? WDYT?

Reply:
Good point. Will make it so.